libs/core/src/lib/datetime-picker/datetime-picker.component.html

```diff
@@ -99,7 +99,7 @@
                             <button
                                 fd-button
                                 fdType="emphasized"
-                                label="Submit"
+                                label="OK"
```

Comment:
Would be great to add `i18` markers for OK and CANCEL buttons

Reply:
Not sure if this has changed but I don't think we usually place i18n markers for core components.

Reply:
Yeah, that can be done by using introducing date-time-i18n service, but not in scope of this pr